docs/video-miners/getting-started/roles-and-responsibilities.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Roles and Responsibilities
+---
+
+# Roles & Responsibilities
```

Comment:
@yondonfu thoughts on moving the page into `core-concepts` section? Thinking about the divio framework, it recommends not using guides and tutorials to explain things at length and rather linking to explanations if they’re important. Your call.

Reply:
Good point. What I think I'll do is:

- Change "Roles & Responsibilities" to "Choosing a Role" which will just give an overview on how to think about choosing your role (i.e. orchestrator vs. transcoder). This has been a common question in Discord so I think having this be the first part of the video miner journey would be helpful
- This updated getting started section will link to a "Roles & Responsibilities" section in core concepts which breaks down the different types of video miners in more detail

Reply:
Fixed in https://github.com/livepeer/livepeer-org/pull/89/commits/9aae965086d48f3a8be62a3118082fe281d669b1